source/MaterialXView/Viewer.cpp

```diff
@@ -255,12 +255,12 @@ Viewer::Viewer(const std::string& materialFilename,
 #endif
 
     // Initialize image handler.
-#if MATERIALX_BUILD_OIIO
-    mx::ImageLoaderPtr imageLoader = mx::OiioImageLoader::create();
-#else
     mx::ImageLoaderPtr imageLoader = mx::StbImageLoader::create();
-#endif
     _imageHandler = mx::GLTextureHandler::create(imageLoader);
+#if MATERIALX_BUILD_OIIO
```

Comment:
OCIO has trouble with some png files for some reason. Setting 2 loaders was "lost" logic at some point -- restoring it.